### PR TITLE
Remove old links, add more links

### PR DESCRIPTION
**Pull request by mjtorsiello**
Removing specific company branding - neutral community site for UrbanCode Velocity

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,8 +1,5 @@
 NOTE: This content is geared towards community engagement. Please refer to version specific product documentation for details.
 
-#### [>> Plugin Documentation](https://www.urbancode.com/plugins/?search=&product_filter%5B%5D=811)
```
Comment (Daniel-Trowbridge) [inline index.md:3]: #### [>> Plugin Documentation](https://www.urbancode.com/plugins/?search=&product_filter%5B%5D=811)

File: index.md [hunk 2]
```diff
@@ -1,8 +1,5 @@
 NOTE: This content is geared towards community engagement. Please refer to version specific product documentation for details.
```
Comment (daniel-trow) [inline index.md:2]:
```suggestion
#### [>> Plugin Documentation](https://www.urbancode.com/plugins/?search=&product_filter%5B%5D=811)
```

File: index.md [hunk 3]
```diff
@@ -1,8 +1,5 @@
...
-
 #### [>> Plugin Documentation](https://www.urbancode.com/plugins/?search=&product_filter%5B%5D=811)
```
Comment (daniel-trow) [inline index.md]:
```suggestion

#### [>> Plugin Documentation](https://www.urbancode.com/plugins/?search=&product_filter%5B%5D=811)
```

**Comment by daniel-trow** [comment]
We want to make sure official documentation is directly accessible so we'll keep these links for now. We'll probably revisit this soon.

File: index.md (hunk 1 above)
Comment (daniel-trow) [inline index.md:3]: The plugin documentation link is still relevant and cannot be directly accessed from the product. Seems like it's still good to include.

File: index.md [hunk 4]
```diff
@@ -1,8 +1,5 @@
 NOTE: This content is geared towards community engagement. Please refer to version specific product documentation for details.
 
-#### [>> Plugin Documentation](https://www.urbancode.com/plugins/?search=&product_filter%5B%5D=811)
-#### [>> HCL Documentation](https://urbancode.hcldoc.com/)
-#### [>> IBM Documentation](https://www.ibm.com/support/knowledgecenter/SSCKX6)
```
Comment (daniel-trow) [inline index.md:5]: Should we keep a direct link to the KC or just refer to the in-product link?

**Pull request merged**